aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java

```diff
@@ -656,6 +657,26 @@ public void close()
          */
         public static final int FILE_SYNC_LEVEL_DEFAULT = 0;
 
+        /**
+         * {@link TimerServiceSupplier} to be used for creating the {@link TimerService} used by consensus module.
+         */
+        public static final String TIMER_SERVICE_SUPPLIER_PROP_NAME = "aeron.cluster.timer.service.supplier";
+
+        /**
+         * Name of the {@link TimerServiceSupplier} that creates {@link TimerService} based on the timer wheel.
+         */
+        public static final String TIMER_SERVICE_SUPPLIER_TIMER_WHEEL = "TimerWheelTimerService";
+
+        /**
+         * Name of the {@link TimerServiceSupplier} that creates a sequence-preserving {@link TimerService}.
+         */
+        public static final String TIMER_SERVICE_SUPPLIER_SEQUENTIAL = "SequentialTimerService";
```

Comment:
PriorityHeap?

Reply:
Renamed the class `SequentialTimerService` to `PriorityHeapTimerService`.